### DISCUSSION TITLE
✅ 🔥 Elementor v3.26 Beta Release 🔥

### DISCUSSION THREAD

**Discussion by avivu** [elementor/elementor#29403] [Announcement]
===========
# [Beta Version Updates - Please Read Below!](https://github.com/orgs/elementor/discussions/29403#discussioncomment-11544349)
===========

<img width="1200" alt="cover-3 26" src="https://github.com/user-attachments/assets/75ad7711-724b-4a6b-af61-00c2a51bbf71">

&nbsp;

**Hey Beta testers,** 👋

We’re excited to announce that Elementor 3.26 Beta is now live! This release builds upon our ongoing performance improvements with further CSS and DOM optimizations, introduces a highly requested enhancement to Loop Grids, and a new AdBlock detection trigger.

Make an impact by testing Elementor 3.26 beta and sharing your impressions on the latest additions, including:

- **[Pro] Dynamic Content for the Off-Canvas widget in a Loop Grid**
- **Improved CSS Loading and Optimized DOM**
- **[Pro] AdBlock Detection Trigger for Popups**
- **[AI] Unify Product Images**
- & More!

Looking forward to hearing your thoughts and feedback on this update!

&nbsp;

# [Pro] New: Dynamic Off-Canvas for Loop Grids

![Off-Canvas Loop Grids](https://github.com/user-attachments/assets/81d74459-0330-443f-b176-d0d9e2472a78)

Elementor 3.26 introduces the ability to add an Off-Canvas widget from  Loop Grid templates, a highly requested feature that adds a new level of interactivity and design flexibility to loop items. This enhancement allows you to reveal additional content dynamically for each specific loop item with an off-canvas panel that opens on click.

### How it works

You can use this feature to display dynamic elements directly to a loop item, which then triggers an Off-Canvas panel with extended content. For instance, in an eCommerce layout, a loop item might display a product image and brief description. With this feature, clicking a button such as “Quick View” or “Learn More” from the Grid, will open an Off-Canvas modal to reveal extra details such as a full product description, pricing, and an “Add to Cart” button - all dynamically linked to the specific item in the loop.

This flexibility allows for more engaging, content-rich experiences in product grids, blog post archives, or service listings, giving site visitors a quick way to access additional information without leaving the main loop view.

#### How to Test It:

1. Create a new page and add the Loop Grid widget to the page.
2. Click on “Create Template” to start creating the loop.
3. On the Loop Template, add any relevant widgets, such as Product Title, Product Price, and Add To Cart, to create a dynamic loop product grid.
4. While editing the loop template, insert the Off-Canvas widget and start adding dynamic content inside the Off-Canvas area to further enrich the loop item with more dynamic widgets, such as Short Description, Additional Information, Product Rating, and more.
5. Add a button widget to the loop item, rename it to “Learn More,” and create a connection between the button link and the previously created Off-Canvas by navigating to the button link settings and choosing the off-canvas dynamic tag.
6. Click on the “Save & Back” button to finish editing the loop template.
7. You will notice the Loop Grid is now populated with all products based on the loop template design.
8. Click on Publish, and check the loop on the front end.
9. Click on the “Learn More” button to reveal the additional content and ensure the Off-Canvas is displaying the dynamic content based on the loop item.
10. Ensure everything is working as expected in the editor and front end.

&nbsp;

# Enhanced Performance: Improved CSS Loading and Optimized DOM

![Enhanced Performance](https://github.com/user-attachments/assets/7f30dfc8-d86a-4e85-bc2b-63f0c563edc1)

Elementor 3.26 introduces significant advancements in performance by further refining how CSS is loaded and optimizing the DOM structure of your pages. These updates ensure your site only loads the styles it needs, while reducing unnecessary HTML elements, making rendering faster and more efficient. With conditional Swiper styles and simplified DOM structures, your site achieves improved Lighthouse scores, quicker page loads, and an overall smoother user experience.

### Continuously Improved CSS Loading

Since version 3.3, Elementor has cumulatively reduced CSS file sizes significantly, cutting down on unused styles and ensuring faster page load times. This initiative has steadily reduced unnecessary CSS, creating faster load times and a leaner structure for Elementor pages, resulting in better performance scores and smoother experiences for site visitors.

#### Key Updates in 3.26

- **Conditional Swiper Styles:** Swiper styles now load only when a carousel is in use, which saves an additional 26.7KB and reduces unnecessary CSS loading for non-carousel pages.
- **Reduced frontend.min.css Size:** The frontend.min.css file has been further reduced, contributing to faster rendering and improving the overall website performance, as only essential styles are loaded.

### Optimized DOM Size

To improve site performance, Elementor 3.26 introduces several optimizations to reduce the size of the DOM, which plays a significant role in page rendering speed. With this release, we’ve simplified the DOM structure to make rendering faster and more efficient, creating a smoother user experience and improving performance scores.

#### Key Improvements:

- **Removal of inner div wrappers:** Unnecessary div wrapper within most Elementor widgets have been removed, helping to reduce the overall DOM complexity. For widgets where the inner <div> is essential (Spacer, Menu, Loop, TOC, Form, Hotspot), the wrapper remains in place. This feature can be activated from Elementor > Settings > Features. Currently in Alpha status and should not be activated on production sites.
- **Replacement of screen-reader elements:** Hidden div elements used for screen readers have been replaced with aria-label attributes, reducing the DOM size while retaining accessibility.
- **Streamlined Carousel Markup:** Outer div wrapper for carousel elements have been removed, simplifying the structure and reducing render time. This feature can be activated from Elementor > Settings > Features. Currently in Alpha status and should not be activated on production sites.

&nbsp;

# [Pro] AdBlock Detection Trigger for Popups

![AdBlock](https://github.com/user-attachments/assets/40aca4da-3f84-4044-86e6-93f4b29e691e)

With Elementor 3.26, managing how your website interacts with ad-blocking visitors becomes more intuitive and effective. A new AdBlock Detection trigger, available in the Display Conditions panel, allows you to display specific popups when visitors are using ad-blocking extensions. This feature helps you communicate important messages, such as encouraging users to disable their ad blocker for a better experience or to access specific content.

To further enhance this functionality, we’ve added the Reload Page dynamic tag. This enables you to create interactive and actionable popups, such as one with a button labeled “I turned off AdBlock.” When clicked, the button refreshes the page, ensuring the updated experience reflects the visitor's actions. Together, these updates make it easier to engage with ad-blocking users in a seamless and user-friendly way.

#### How to Test It:

1. Create a new popup.
2. Design your popup with content related to AdBlock detection. Ensure it includes a button or any link.
3. On the link, open the dynamic tags window. Under Actions, choose Page Reload.
4. Click the Publish button and set the conditions.
5. Choose where to display the popup.
6. Under Triggers, activate the AdBlock Detection option and set the delay time for when the popup should appear.
7. Click on Save & Close.
8. Navigate to any page and check it live to ensure the popup does not appear initially.
9. Go to your browser extensions and install an AdBlock extension.
10. After installing and activating the extension, refresh the previous page. At this point, the popup should appear because an active AdBlock extension is detected.
11. After the popup appears, keep it visible and deactivate the AdBlock extension.
12. Click on the link with the dynamic tag, ensuring the page reloads after the click. Confirm that the popup does not reappear.
13. Verify that everything is functioning as expected in both the editor and the front end.

&nbsp;

# Unify Product Images with Elementor AI

![unify-ai](https://github.com/user-attachments/assets/9cad7fc3-caaf-4f4d-9575-2f4e3c0daa3f)

The new Unify Product Images feature powered by Elementor AI, helps you achieve a consistent and professional look for your online store. This new feature effortlessly standardizes your product images, ensuring they all have the same style and quality, no matter their origin. This not only saves you valuable time and effort but also elevates your brand image, making your products more appealing to customers.

Access this powerful tool from any Product page or apply it as a bulk action to your entire product catalog. Enhance your WooCommerce store's visual appeal and free yourself from tedious image editing with Elementor AI.

&nbsp;

# Additional Updates

<img width="1200" alt="Additional Updates 3 26" src="https://github.com/user-attachments/assets/467a371d-f74f-49fb-ae3d-84d234c60efe">

&nbsp;

- **Added support to Gutenberg Blocks in the frontend:** The rendering of Blocks is now supported and visible in both the front-end and back-end, but cannot be edited with Elementor.
- **Accessibility enhancements:** Includes aria-label attributes for sliders, role and aria-role description attributes, and improved keyboard navigation. These updates help make Elementor pages with carousels more accessible, ensuring compliance with web accessibility standards and creating a more inclusive user experience.
- **Updated Safari support:** Minimum supported Safari version is now 15.5. This update ensures compatibility with modern features, enhancing stability and performance for visitors on modern versions of the browser.
- **Paragraph spacing control for Text Editor widget:** Provides control over paragraph spacing within the Text Editor widget, allowing you to customize the spacing between paragraphs even when a global paragraph spacing setting is applied. This update gives you greater layout control for text-heavy sections.
- **Hover Color option for links in the Heading widget:** Adds a hover color setting to the Heading widget when using links, enabling hover effects for headings with links, which can help emphasize interactive elements on a page.
- **Space Between Dots” control for Carousel widgets:** New control for customizing the space between pagination dots on carousel widgets, providing flexibility to adjust spacing as needed, especially useful for smaller screen sizes.

### Updated Feature Status

- **Merged Features:**
  - Optimized Control Loading – Merged
  - Upgrade Swiper Library – Merged
- **Upgraded to Stable:**
  - Nested Elements – Updated from beta to stable
- **Status and Activation Change:**
  - Element Caching – Now active for existing users

----

# More Improvements

#### Free version

* New: Unify product images seamlessly with AI
* Tweak: Added "Space Between Dots" control to Image Carousel widget (#2526, #3277, #21697)
* Tweak: Added paragraph spacing control in Text Editor widget (#25431, #20144)
* Tweak: Replaced hidden `elementor-screen-only` div with `aria-label` attributes (#28355)
* Tweak: Added hover and transition color controls for a link in Heading widget (#12877)
* Tweak: Updated Image Spacing control to support only PX units in Image Carousel widget (#21827)
* Tweak: Created CSS variables for Row and Column gaps value in Container (#24178)
* Tweak: Added the ability to set `aria-label` in Image Carousel widget (#28355)
* Tweak: Replaced the Wrap Align Content select control with a choose control in Container (#22640)
* Tweak: Implemented accessibility improvements for pagination bullets in Image Carousel widget (#28674)
* Tweak: Load Nested Elements styles only when they are in use
* Tweak: Added support for rendering Gutenberg blocks on the frontend
* Tweak: Removed `elementor-widget-container` div from Elementor widgets as part of the Optimized Markup experiment
* Tweak: Updated CSS to utilize `inset` CSS logical property in various locations
* Tweak: Updated minimum required Safari version to 15.5
* Tweak: Transition Duration controls are displayed only when a background color is set
* Tweak: Transition Duration controls are displayed only when a border is set
* Tweak: Improved keyboard accessibility for the section and container presets area
* Tweak: Merged "Upgrade Swiper Library" feature to version
* Tweak: Merged "Optimized Control Loading" feature to version
* Tweak: Activated "Element Caching" feature for existing sites
* Tweak: Promoted "Nested Elements" feature to Stable status
* Tweak: Removed `elementor/core/schemes`
* Tweak: Updated `eicons` library to v5.33.0
* Fix: Container Gap values not transferring correctly between sites when using Cross-site Copy and Paste (#24111, #28343)
* Fix: Removed anchor scroll warnings from the console (#29199, #29350)
* Fix: PHP error log appearing due to the Usage Data Sharing cron job (#29153)
* Fix: Deprecated `print_emoji_styles` warning in Theme Builder screen (#27502)
* Fix: Display Condition feature is available without an active Elementor license in Floating Elements
* Fix: "Generate with Elementor AI" button is missing in the Media Library list mode
* Fix: Resized file size exceeding the maximum limit does not display an error in AI Image
* Fix: Images from URL incorrectly open the Generate modal

#### Pro version

* New: Introducing dynamic content support for Off-canvas in Loop Grid widget templates - enabling seamless display of additional content dynamically (#29049, #28935, #7106)
* New: AdBlock Detection trigger for Popups - display targeted popups for visitors using ad blockers
* New: Added Reload Page link action for Dynamic Tags
* Tweak: Added "Space Between Dots" control to Carousel widgets (#2526, #3277, #21697)
* Tweak: Replaced hidden `elementor-screen-only` div with `aria-label` attributes (#28355)
* Tweak: Removed `elementor-widget-container` div from Elementor widgets as part of the Optimized Markup experiment
* Tweak: Optimize popup styling by loading it only when necessary
* Tweak: Updated Gap between slides control to support only PX units in Carousel widget
* Tweak: Removed the dependency between the Tabs handler and the Menu handler
* Tweak: Added the ability to set `aria-label` in Carousel widgets
* Tweak: Added `aria-roledescription=carousel` to carousel wrapper
* Tweak: Added `aria-roledescription=slide` to slide wrapper
* Tweak: Optimized carousel widgets markup as part of the Optimized Markup experiment
* Tweak: Updated minimum required Safari version to 15.5
* Tweak: Facebook widgets are no longer supported on mobile due to Facebook functionality changes
* Fix: Motion Effects sticky feature is not working when Scroll Snap is enabled
* Fix: Custom Image Resolution option is not working in Gallery widget
* Fix: 'Reading Settings' unexpectedly reduces the number of posts displayed when using the Archive with Pagination setting
* Fix: Step Form does not scroll to the top when the first field is not an input field in Form widget
* Fix: Display Condition feature is available without an active Elementor license in Floating Elements
* Fix: Background options of Flip Box in Loop Grid widget are not working when images are added using ACF Dynamic Tags
* Fix: Dropdown icon in the Select field of the Form widget is cut off when the Inline Font Icons feature is activated
* Fix: Incompatibility between Form widget and WPML when using Nested Elements
* Fix: Performance issues when changing the title name in the Menu widget
* Fix: Warnings appear when Element Caching is set to Inactive in Carousel widgets

----

## How to Install the Beta Version

If you still haven't enabled beta testing in Elementor, please follow these steps:

1. Go to Elementor > Tools > Versions
2. Enable the beta tester feature
3. Go to Dashboard > Updates and update to the latest beta version.
4. This will get you set up to test the new Elementor 3.26 Beta.

## Notes

As we make changes to our GitHub in an effort to keep it organized, improve the way we deliver information to you, and our ability to review your feedback, please follow the instructions below when reporting bugs, or making feature requests:

- To report a bug related to beta versions - open a new [🐞 Bug Report](https://elemn.to/gh-new-bug-report)
- For feature requests - open a new [🚀 Feature Request](https://elemn.to/gh-new-feature-request)
- For other comments specifically about this beta version - comment here

Designated time for release: `December 16th, 2024 (Estimated)`

Many thanks for your support and help!

Cheers,
_Elementor Team_

**Comment by KingGrizzly** [comment]
Cross-site pasting gap bug is getting fixed. Woo hoo! Thanks!

**Comment by Flickimp** [comment]
Uploading Screen Recording 2024-11-25 at 16.34.28.mov…

Does anyone else find the Off Canvas in the loop go through all of the posts?
I have 3 posts in the Loop Grid. 
When I open the Off Canvas I have to go through all 3 as they all returned dynamic data.

So rather than it only showing me the post details I clicked - it shows them all.

This happens for Posts and Products.

**Reply by nicholaszein** [reply]
Hi @Flickimp!

I'm testing the Off-Canvas in the Loop right now, and it's working as expected. Can you elaborate a little? Maybe share some steps to reproduce so we can try on our end? Maybe we missed something.

**Reply by Flickimp** [reply]
It's fine now.
I suspect a glitch on my side as I reset my WP dummy site and started again.
I'll be doing a video later to release.

**Reply by nicholaszein** [reply]
Thanks for updating us @Flickimp! 🙏 

We were suspecting that somehow your Elementor Pro version wasn't updated yet. 😅

**Comment by midihead7** [comment]
In Elementor Pro 3.26.0-beta1, it appears that the freezing issue when editing the Title field in the Menu widget (**Performance issues when changing the title name in the Menu widget**) has finally been fixed. I'm no longer experiencing freezing when editing the Title field and saving the changes.

HOWEVER, the developers have only fixed half of the Menu freezing issues that I reported in my ticket to them (Case #TSJG9TJF14AK) nearly a year ago. I had also said that the **URL field** was also causing freezing when the Menu URL field was edited. **This URL freezing issue still exists on Elementor Pro 3.26.0-beta1**. I just tested and I still see the same freezing and high CPU usage when typing in the Menu URL field and saving takes a long time after editing the URL field. Here is a screenshot showing that the CPU usage is still high after keypresses in the URL field:

![Elementor 3 26 Beta Menu URL Keypress CPU](https://github.com/user-attachments/assets/c434d246-1cb0-469e-b0fa-3e2c4f0c3a01)

I have also updated the ticket to inform Tier 3 Support that the Menu URL Freezing issue still exists. Please address and fix the Menu URL freezing so that I can finally move on from these performance issues. Thank you.

**Reply by nicholaszein** [reply]
Hi @midihead7!

Thank you for your detailed feedback! We're glad to see the problem with the Title field as been resolved. 💪 

We are surely going to verify your report about the URL! 🕵️ 

Kind regards

**Reply by efintina** [reply]
Similar issue when changing title, icon, or CSS ID in **Tabs**: performance issues, including very slow saving.

**Comment by Flickimp** [comment]
In case anyone wants to see a Video Review of the Dynamic Off Canvas.
[https://youtu.be/bERknoxbvno?feature=shared](https://youtu.be/bERknoxbvno?feature=shared)

**Comment by datgausaigon** [comment]
Hi, 
How about this `2 years` old Elementor `Pro` issue https://github.com/elementor/elementor/issues/20769 ?
Thanks

**Comment by JoeCrescoll** [comment]
Great news !

Are there any plans to add the Dynamic Off-Canvas functionality to the Loop Carousels too ?

**Reply by nicholaszein** [reply]
Hi @JoeCrescoll!

Unfortunately, we can't add the Off-Canvas to Loop Carousels due to the nature of the widget. Carousels in general duplicate and delete content dynamically, which in turn would create a very poor experience for the users, not to mention having to include intricate scripts to handle the triggers. So we don't have plans to include it in the Loop Carousel.

**Comment by RonR-WebDesign** [comment]
Bug(?).   Create a Loop Grid and added to loop template, a post title and post content.   Added Off-Canvas in the loop template and again added post title and post content.  The post content will not appear.  If I remove post content from the loop template then it appears.

**Reply by nicholaszein** [reply]
Hi @RonR-WebDesign,

In WordPress you can't have two instances of the Post Content inside a post or a loop. You can have only one. If you use two of them one of the following issues will happen:

1. Only one will be loaded (as you already found out).
2. It will cause something called an **"Infinite Loop"**, where a content is loaded inside a content, that is loaded inside a content, and so on until your site crashes with a fatal error or memory depletion.

In this case there is nothing we can do. You can use only one Post Content widget per Loop Item Template, regardless if it's inside an Off-Canvas or not. I hope you can understand!

**Reply by nicholaszein** [reply]
Thank you for raising this out though! It's a little caveat about WordPress that not everyone knows or understands.

**Reply by RonR-WebDesign** [reply]
But....  the Post Title, Info, and Featured Image all work fine.  It's just Post Content.

**Reply by nicholaszein** [reply]
Yes, these can be repeated. What can't be repeated is the function `the_content`, which is what the Post Content widget is under the hood @RonR-WebDesign. I know, it's a bummer, but this is a WordPress limitation that we can't avoid. I hope you can understand! What you can do is to use the Post Excerpt widget, or a Text Editor widget with the Post Excerpt Dynamic Tag. This should solve your issue! 😉

**Reply by RonR-WebDesign** [reply]
Ok. It all makes sense.  And yes, it works with Post Excerpt.  We can live with that...  and, thanks for the quick responses - much appreciated!

**Comment by avivu** [comment]
Hi all,

We just released Elementor **v3.26 Beta 2**.



The versions include the following:


#### Free version


* Tweak: General improvements in version




#### Pro version


* Fix: Dropdown area is not opening correctly in certain scenarios in Menu widget


–

Thank you!

**Comment by avivu** [comment]
Hi all,

We just released **Elementor v3.26 Beta 3**.


The versions include the following:


#### Free version

* Tweak: Improved behavior and messaging for the warning displayed when navigating between documents 


#### Pro version

* No change


–

Thank you!

**Comment by hitsgee** [comment]
I have noticed a bug with the new update but it's a hard one to recreate. 

I used the off canvas widget styled it all and got it working great. When I then came back to this widget after editing some other pages/templates the settings for the off canvas Widget are blank with no options for style etc.

I tried clearing cache etc but nothing helped. I had to remove the widget and add it all again for it to work but then the same issue when returning later to it. I will try and get a video over later for you to see the issue.

**Reply by efintina** [reply]
I can confirm this issue. It appears randomly and disappears after clearing all the cache (including server cache and transients).

**Reply by doronwolf** [reply]
Could there be a specific step that needs to be taken in order to reproduce this issue? I'm trying to recreate it but have failed so far

**Reply by efintina** [reply]
I'm not sure, but it seems that occur after duplicate an existing off-canvas widget.

**Reply by hitsgee** [reply]
Here is a video of the bug in action.

Weirdly the heading of the panel does not change when I click the "Off Canvas widget"

If I remove that widget and add a new one, it works fine.

[Edit _Team – Loop_ with Elementor (1).webm](https://github.com/user-attachments/assets/ce3fded4-28ce-4170-bb3a-1504e9481c11)

**Reply by hitsgee** [reply]
Console also shows some errors.
![Desktop-screenshot-12-05-2024_12_57_PM](https://github.com/user-attachments/assets/5b975765-5a13-48ce-89a9-fe33d5f6322e)

**Reply by hitsgee** [reply]
If I edit the loop template with the off canvas widget via the loop grid template editor it works perfectly. If I edit the loop template directly via templates it does not work. Very strange.

**Reply by doronwolf** [reply]
thank you @hitsgee - seems like this is the way to reproduce the issue 
meaning: If I try to edit the loop item from the templates (instead of inline editing it from the page that contains the loop) then I'm experiencing the same issue, I'll try to promote a fix for that

**Reply by efintina** [reply]
Maybe, in this way you will solve the same issue that raise for inline editing. I don't use any loop or templates in the off-canvas widget but have the same problem.

**Reply by nicholaszein** [reply]
Hi @efintina, can you elaborate? When you say inline editing, what do you mean?

**Reply by efintina** [reply]
Direct editing of the off-canvas widget. Contains only container, icon, title, HTML. No looping or templates. Sorry for the incorrect phrasing.

**Reply by nicholaszein** [reply]
@efintina,

Just to clarify, you mean that opening a document (any other than a Loop Item Template) that contains an Off-Canvas, and trying to edit it causes the same problem described and recorded by @hitsgee?

Or is there any specific steps to replicate, specific settings, site and server stack, browser or OS that needs to be in use in order to replicate this issue? 

Does it happen in this version of Elementor (3.26.X), or is it something you've been experiencing in the latest stable version of Elementor?

**Reply by efintina** [reply]
For this test, I started with a new page where I added an Off-Canvas, played with the edit mode switch (On/Off) a few times, saved it and exited to wordpress. What happens when I open it again to edit it, you can see below.

It happen with Elementor v3.26.0-beta3 and Elementor Pro v3.26.0-beta2. I tested only on Firefox Developer Edition.


https://github.com/user-attachments/assets/7e0c2b85-6884-4ca2-8a0f-e7f4962747f7

**Reply by nicholaszein** [reply]
Hi @efintina,

I've been trying my best to reproduce your issue with the latest 3.26.0-beta4, and everything seems to be loading correctly. In the video below I tested in multiple different ways. What may I be doing wrong?


https://github.com/user-attachments/assets/891b52ab-8980-4e8f-bfd3-085b5e52a38c

It's important to note that I have multiple plugins active, but no cache. Here's my System Info:



<details>
<summary>Click to reveal</summary>

````txt

					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.27.1
	MySQL version: mariadb.org binary distribution v10.11.10-MariaDB-ubu2204
	PHP Version: 8.1.30
	PHP Memory Limit: 2048M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 2048M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.7.1
	WP Multisite: No
	Max Upload Size: 2 GB
	Memory limit: 1024M
	Max Memory limit: 2024M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: America/Sao_Paulo
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.1.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36

== Active Plugins ==
	Admin and Site Enhancements (ASE)
		Version: 7.6.1.1
		Author: wpase.com

	Advanced Custom Fields PRO
		Version: 6.3.5
		Author: WP Engine

	All-in-One WP Migration
		Version: 7.87
		Author: ServMask

	All-in-One WP Migration Unlimited Extension
		Version: 2.64
		Author: ServMask

	Code Snippets
		Version: 3.6.6.1
		Author: Code Snippets Pro

	Elementor
		Version: 3.26.0-beta4
		Author: Elementor.com

	Elementor Pro
		Version: 3.26.0-beta4
		Author: Elementor.com

	FluentSMTP
		Version: 2.2.83
		Author: FluentSMTP & WPManageNinja Team

	Instant Images
		Version: 7.0.0
		Author: Darren Cooney

	JetEngine
		Version: 3.6.0.1
		Author: Crocoblock

	Query Monitor
		Version: 3.17.0
		Author: John Blackbourn

	Solid Security Basic
		Version: 9.3.6
		Author: SolidWP

	WooCommerce
		Version: 9.4.3
		Author: Automattic


== Settings ==
	
	cpt_support: post, page, product
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap
	meta_generator_tag: 1


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	woocommerce: Active


== Elementor Experiments ==
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active
	Container: Active
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Markup: Active
	Plugin Onboarding: Active by default
	CSS Smooth Scroll: Active by default
	Default to New Theme Builder: Active
	Header & Footer: Active
	Elementor Home Screen: Active by default
	Landing Pages: Inactive by default
	Nested Elements: Active
	Editor Top Bar: Active
	Pages Panel: Inactive by default
	Element Caching: Active by default
	Link In Bio: Active by default
	Floating Buttons: Active by default
	Elementor Editor Events: Inactive by default
	Atomic Widgets: Inactive by default
	Global Classes: Inactive by default
	Launchpad Checklist: Inactive by default
	Menu: Active


== Log ==
	
Log: showing 20 of 362024-12-11 12:01:48 [info] Elementor/Upgrades - _v_3_26_0 Start  
2024-12-11 12:01:48 [info] Elementor/Upgrades - _v_3_26_0 Finished 
2024-12-11 12:01:48 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.26.0-beta2',
  'to' => '3.26.0-beta3',
)]
2024-12-11 12:02:30 [info] elementor-pro::elementor_pro_updater Started 
2024-12-11 12:02:30 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-12-11 12:02:30 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-12-11 12:02:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.26.0-beta2',
  'to' => '3.26.0-beta3',
)]
2024-12-11 12:02:30 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.26.0-beta2',
  'to' => '3.26.0-beta3',
)]
2024-12-11 12:05:51 [info] elementor-pro::elementor_pro_updater Started 
2024-12-11 12:05:51 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-12-11 12:05:51 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-12-11 12:05:51 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.26.0-beta3',
  'to' => '3.26.0-beta4',
)]
2024-12-11 12:05:51 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.26.0-beta3',
  'to' => '3.26.0-beta4',
)]
2024-12-11 12:06:30 [info] elementor::elementor_updater Started 
2024-12-11 12:06:30 [info] Elementor/Upgrades - _on_each_version Start  
2024-12-11 12:06:31 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.26.0-beta3',
  'to' => '3.26.0-beta4',
)]
2024-12-11 12:06:32 [info] Elementor/Upgrades - _on_each_version Finished 
2024-12-11 12:06:32 [info] Elementor/Upgrades - _v_3_26_0 Start  
2024-12-11 12:06:32 [info] Elementor/Upgrades - _v_3_26_0 Finished 
2024-12-11 12:06:32 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.26.0-beta3',
  'to' => '3.26.0-beta4',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

If there is something else I should do to try to reproduce your issue, please let me know!

Regards

**Reply by doronwolf** [reply]
Hi @hitsgee I believe we found the root cause and working on a fix
In the meantime, can you check if deactivating the Optimized markup experiment solves the issue for you?
(@nicholaszein ..this might also be relevant to @efintina ...but I'm just guessing)

**Reply by efintina** [reply]
Due to the inconsistent occurrence of the strange behavior mentioned, it is difficult to say whether Optimized Markup Experiment is indeed responsible for it. However, I can confirm that after disabling it, the error no longer appeared. 
I have tested this on several websites/servers with different configurations.

**Comment by avivu** [comment]
Hi all,

We released **Elementor v3.26 - Beta 4**.


The versions include the following:


#### Free version

* Fix: Unify product images intro popover is not visible
* Fix: Elementor supported post types are being overridden after a WooCommerce product is created
* Fix: YouTube video in lightbox is not presented as expected in Video widget


#### Pro version

* Fix: Popup overlay shown for non-logged users even when disabled
* Fix: Accessible navigation in popups caused unintended scrolling of the page when triggered


–

Thank you!

**Comment by toineenzo** [comment]
With the latest beta I get a lot of errors when clicking the advanced tab inside an element. This does not happen with container but a lot with the Heading and text editor widget. The advanced tab stays blank and the following error is shown in the console.
```
TypeError: this.getEditorControlModel(...) is undefined
    getEditorControlView editor-modules.min.js:3
    C common-modules.min.js:2
    highlightIconIfFilled display-conditions.min.js:2
    Backbone 4
    openActiveSection editor.min.js:3
    onRenderCollection editor-modules.min.js:3
    Backbone 11
    setPage editor.min.js:3
    renderTab editor.min.js:3
    activateTab editor.min.js:3
    registerTabRoute editor.min.js:3
    run web-cli.min.js:4382
    to web-cli.min.js:6753
    route web-cli.min.js:3884
    apply editor.min.js:3
    run web-cli.min.js:8190
    runInstance web-cli.min.js:4398
    run web-cli.min.js:4388
    run web-cli.min.js:3862
    updatePanelPage editor.min.js:3
    get editor.min.js:3
    apply editor.min.js:3
    run editor-document.min.js:2
    runInstance web-cli.min.js:4398
    run web-cli.min.js:4388
    run web-cli.min.js:3862
    apply editor.min.js:3
    apply editor.min.js:3
    run editor-document.min.js:2
    runInstance web-cli.min.js:4398
    run web-cli.min.js:4388
    run web-cli.min.js:3862
    onEditRequest editor.min.js:3
    Backbone 4
    onEditButtonClick editor.min.js:3
    onClickEdit editor.min.js:3
    Backbone 2
```

I also believe there are still some critical bugs with the dynamic off canvas function, but i unfortunately havent got the time to test this beta thoroughly.

**Reply by toineenzo** [reply]
Let me correct myself, this didn't start happening with the latest beta, but happens when the Optimized markup experiment is enabled.

**Reply by doronwolf** [reply]
Hi @toineenzo it seems some 3rd party plugins don't play well with the Optimized markup experiment, can you confirm that the issue is gone if you disable those plugins? 
(On our side, we are working on communicating the required changes that 3rd party needs to take in order to work with this experiment)

**Reply by toineenzo** [reply]
> Hi @toineenzo it seems some 3rd party plugins don't play well with the Optimized markup experiment, can you confirm that the issue is gone if you disable those plugins? (On our side, we are working on communicating the required changes that 3rd party needs to take in order to work with this experiment)

@doronwolf 
I only had ACF and ASE enabled, but for some reason its working again now.
Let me try to test the other aspects of this beta and ill get back with my findings

**Comment by avivu** [comment]
Hi all,

We just released **Elementor v3.26 Beta 5**


The versions include the following:


#### Free version

* Fix: Swiper styling missing from Lightbox inside Gallery widgets



#### Pro version

* Fix: Dynamic Off-Canvas widget is not editable from Elementor Theme Builder
* Fix: Swiper styling missing from Lightbox inside Gallery widgets



–

Thank you!

**Comment by AIBCREER** [comment]
You say you are going to update Nested elements from beta to stable, but people keep complaining about performance issues on Facebook and here. 
I too find that with just a few nested elements in the elementor editor it becomes very uncomfortable to work with. I follow all the performance recommendations from Elementor and it doesn't help. My computer is only running at about 50% when Elementor starts to die.

**Reply by KingGrizzly** [reply]
I have very little trouble on Mac OS/Chrome but my co-worker on a PC and client are finding some pages unusable due to nested widgets and the memory leak issue. For example, a slider inside of a tab widget grinds their editing experience to a halt.

**Reply by KingGrizzly** [reply]
Actually, the tab widget becomes unusable when nesting a basic container with an image and text under each tab. After a few tabs are added, the left sidebar starts to lag to the point where trying to add an icon to the tab might take 5 minutes to load the media library. It gets so bogged down the page is unusable.

**Reply by nicholaszein** [reply]
Thank you for the feedback guys. Unfortunately, we don't have the same experience on our end while testing on different browsers, machines, and environments, including with Addons and 3rd-party themes.

But we are going to take a look on those. It would help if you submit new detailed bug reports so we can look closer at your site stacks and see if we can find the root of the issue on your sites.

For now, we are closing this thread.

Kind regards

**Comment by nicholaszein** [comment]
Hi Beta testers! 👋 

Hey everyone!

### Elementor and Elementor Pro version 3.26.0 were released to the public! 🚀 🥳 🙌 

We'd like to thank you all who contributed to this discussion. 🙏 

Special thanks go to our contributors who helped make this release possible:

- @Flickimp 
- @midihead7 
- @efintina 
- @RonR-WebDesign 
- @hitsgee 
- @toineenzo 

Thank you for your feedback! See you on the next one! 💪 😉 

**Cheers 🥂**

> Changelogs of Elementor Core and Pro
> [Elementor Core](https://wordpress.org/plugins/elementor/#developers)
> [Elementor Pro](https://elementor.com/pro/changelog/)
